src/test/java/com/example/TestUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.example;
+
+import com.example.weatherwaveapi.model.response.openweather.WeatherOpenApiContainer;
+import com.example.weatherwaveapi.model.response.openweather.forecastapi.ForecastDataContainer;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.City;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.SunActivityInfo;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.WeatherMetrics;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.weather.Weather;
+
+import java.util.List;
+
+public class TestUtil {
+    public static final String URL = "http://localhost:8080/weather/city=london";
+    public static final String CITY_LONDON = "london";
+    public static final Integer ZIPCODE = 94040;
+    public static final String COUNTRY_US = "US";
+    public static final double TEMPERATURE = 35D;
+    public static final int VISIBILITY = 10000;
+    public static final String COUNTRY_GB = "GB";
+    public static final String DESCRIPTION = "light intensity drizzle";
```

Comment:
Please, use more meaningful names. As I understand TestUtil class is one for all tests and when you will try to use it in other tests such variables can cause misunderstanding and unpredictable results